vyper/builtin_functions/functions.py

```diff
@@ -1264,14 +1264,18 @@ class Send(BuiltinFunction):
 
     _id = "send"
     _inputs = [("to", AddressDefinition()), ("value", Uint256Definition())]
+    _kwargs = {"gas": KwargSettings(Uint256Definition(), "gas")}
```

Comment:
I think the default should be something semantically meaningful here, like 0

Reply:
changed